test/index.test.js

```diff
@@ -224,6 +226,26 @@ describe('glob2base test patterns', function () {
 
     done();
   });
+
+  it('should not increase calc. time exponentially by \'/\' count [CVE-2021-35065]', function (done) {
```

Comment:
Comment says "exponential". This regex will exhibit polynomial growth, not exponential growth.

Reply:
Got it. I'll modify it.

Reply:
I'll remove this case and add a new case to check absolute time.